pages/donghuas/_donghua.vue

```diff
@@ -10,24 +10,40 @@
             <h2>{{ donghua.title }}</h2>
             <div>{{ donghua.description }}</div>
             <div>
-              Genres:
+              <h3>Genres:</h3>
               <a-tag v-for="genre in donghua.genres" :key="genre">{{
                 genre.name
               }}</a-tag>
             </div>
             <div>
-              Tags:
+              <h3>Tags:</h3>
               <a-tag v-for="tag in donghua.tags" :key="tag">{{
                 tag.name
               }}</a-tag>
             </div>
+            <div>
+              <h3>Popularity:</h3>
+              <p v-for="popularity in donghua.popularity" :key="popularity">
+                {{ popularity.platform }}:
+                <a-tag>{{ popularity.numUsers }}</a-tag>
+              </p>
+            </div>
+            <div>
+              <h3>Score:</h3>
+              <p v-for="score in donghua.score" :key="score">
+                {{ score.platform }}:
+                <a-tag>{{ score.score }}</a-tag>
+              </p>
+            </div>
           </div>
         </a-col>
       </a-row>
       <a-tabs default-active-key="1">
         <a-tab-pane key="1" tab="Media"></a-tab-pane>
         <a-tab-pane key="2" tab="Related Works"></a-tab-pane>
-        <a-tab-pane key="3" tab="Staff"></a-tab-pane>
+        <a-tab-pane key="3" tab="Staff">
+          <h4 v-for="staff in staffs" :key="staff">{{ staff.name }}</h4>
```

Comment:
Need to also include the name of the person who has the role. Maybe use a table for this? antd vue has a built-in component for this: https://www.antdv.com/components/table/

Reply:
But the stuff.name containe both role and name as 
![image](https://user-images.githubusercontent.com/6095752/117564141-2a5e0d80-b0dd-11eb-9c31-f68147300230.png), it is not a normal format?

Reply:
oh yeah I think it should be renamed `staffRole` not `staff`. the name is for the role name only (ie in this case the role is for Ye Xiu's VA)

Reply:
Okay I will fix it later